content/1.7/back-office/international/locations/countries/add-edit-country.md

```diff
@@ -0,0 +1,74 @@
+
+     - **Customer** -  the following parameters can be added to the address format: lastname, firstname, birthday, email, website, company, siret, reset_password_token, htmlFields. 
+      - **Warehouse** - the following parameters can be added to the address format: reference, name, management_type, htmlFields
+     - **Country** - the following parameters can be added to the address format: iso_code, name, zip_code_format, htmlFields
+     - **State** - the following parameters can be added to the address format: iso_code, name, htmlFields 
+     - **Address** - ithe following parameters can be added to the address format: company, lastname, firstname, address1, address2, postcode, city, other, phone, phone_mobile, vat_number, dni, htmlFields.
+When a parameter is already existing, when trying to save it, the error shows: "This key has already been used.: phone"
+     
+ Below there are 4 buttons with different actions:
+ 
+     - **Use the last registered format** - When this button is pressed, the following modal is displayed: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".
+When pressed "ok" will input the address format box with the last registered formats.
+     - **Use the default format** - When this button is pressed, the following modal is displayed: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".
+When pressed "ok" will input the address format box with the default address formats.
+     - **Use my current modified format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".
+     - **Clear format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel". When "ok" is pressed, all the fields are removed from the address format.
+     
+ - **Active** - This toggle can be turned to "Yes" or "No" to enable or not the country. By default, it is set to "yes". A disabled country will not be suggested as an option when a visitor wants to create or edit an address. The tooltip displays: "Display this country to your customers (the selected country will always be displayed in the Back Office).
+ - **Contains states** - This toggle indicates whether the country has "states" or not. It can be turned to "Yes" or "No". By default, it is set to "no". If set on "yes", a new field allowing to select the state is displayed in the address form.
+ - **Do you need a tax identification number?** - This toggle can be turned to "Yes" or "No". By default, it is set to "no".
+ - **Display tax label (e.g. "Tax incl.")** - This toggle allows choosing whether the tax status (included or excluded) should be displayed next to the prices or not. It can be turned to "Yes" or "No". By default, it is set to "yes".
```

Comment:
```suggestion
 - **Display tax label (e.g. "Tax incl.")** - This toggle button allows choosing whether the tax status (included or excluded) should be displayed next to the prices or not. It can be turned to "Yes" or "No". By default, it is set to "Yes".
```